tests/end_to_end/multi-switch/irix-o2-noswitch-out.c

```diff
@@ -11,9 +11,9 @@ s32 test(s32 arg0) {
     phi_a0_3 = arg0;
     phi_a0_4 = arg0;
     phi_a0_5 = arg0;
-    if (arg0 >= 0x33) {
-        if (arg0 >= 0x6C) {
-            if (arg0 != 0xC8) {
+    if (arg0 >= 0x00000033) {
+        if (arg0 >= 0x0000006C) {
+            if (arg0 != 0x000000C8) {
```

Comment:
These tests have `--zfill-constants` enabled, which is why they have more zeros. (otherwise, this isn't normal)